src/style.css

```diff
@@ -1,3 +1,68 @@
+* {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  position: relative;
+}
+
+.logo {
+  padding-top: 2px;
```

Comment:
this padding is redundant. Make your image as `block` element using `display:block`
```suggestion
```